python/src/load.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
+#include <iostream> // TODO
```

Comment:
```suggestion
#include <iostream> // TODO
```

I think we can remove this now